src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -375,6 +375,7 @@ class HuiMapCard extends LitElement implements LovelaceCard {
 
       #root {
         position: relative;
+        height: 100%;
       }
 
       :host([ispanel]) #root {
```

Comment:
Remove this rule as it is not needed anymore now.

Reply:
But that is actually the fix itself for the map problem that I am trying to solve.
Or do you mean the whole `#root` rule?

Reply:
I mean the `:host([ispanel]) #root` rule, that is duplicate now as it is already in the `#root` rule